test/functional/blockchain.py

```diff
@@ -79,5 +82,11 @@ def _test_getblockheader(self):
         assert isinstance(int(header['versionHex'], 16), int)
         assert isinstance(header['difficulty'], Decimal)
 
+    def _test_getdifficulty(self):
+        res = self.nodes[0].getdifficulty()
```

Comment:
I'd prefer a more descriptive name than `res`, perhaps `difficulty`? (or even combine with the line below)